examples/dtls-echo/Makefile

```diff
@@ -49,6 +49,8 @@ CFLAGS += -DDEVELHELP
 # NOTE: Add the package for TinyDTLS
 USEPKG += tinydtls
 
+USEMODULE += tinydtls
```

Comment:
Yes, as far as I understood #5891 the idea now:

* `USEPKG` just is responsible for *getting* the package, while
* `USEMODULE` is responsible for *building* it as a module.